shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1238,13 +1240,24 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
     }
 
     @Data
+    @AllArgsConstructor
     public static final class MergeSegments implements Request, WireCommand {
         final WireCommandType type = WireCommandType.MERGE_SEGMENTS;
         final long requestId;
         final String target;
         final String source;
         @ToString.Exclude
         final String delegationToken;
+        final List<ConditionalAttributeUpdate> attributeUpdates;
```

Comment:
FYI this would have required bumping the Wire Protocol version. However we have already bumped it for release 0.10, so no need to do it again. But in the future any such changes to the wire format would require that.

Reply:
Ok, got it.